definitions/npm/react-native-vision-camera_v2.x.x/flow_v0.142.x-/react-native-vision-camera_v2.x.x.js

```diff
@@ -0,0 +1,1358 @@
+    | "device/focus-not-supported"
+    | "device/camera-not-available-on-simulator";
+  declare type FrameProcessorError = "frame-processor/unavailable";
+  declare type FormatError =
+    | "format/invalid-fps"
+    | "format/invalid-hdr"
+    | "format/invalid-low-light-boost"
+    | "format/invalid-format"
+    | "format/invalid-color-space"
+    | "format/invalid-preset";
+  declare type SessionError =
+    | "session/camera-not-ready"
+    | "session/audio-session-setup-failed"
+    | "session/audio-in-use-by-other-app"
+    | "session/audio-session-failed-to-activate";
+  declare type CaptureError =
+    | "capture/invalid-photo-format"
+    | "capture/encoder-error"
+    | "capture/muxer-error"
+    | "capture/recording-in-progress"
+    | "capture/no-recording-in-progress"
+    | "capture/file-io-error"
+    | "capture/create-temp-file-error"
+    | "capture/invalid-video-options"
+    | "capture/create-recorder-error"
+    | "capture/recorder-error"
+    | "capture/no-valid-data"
+    | "capture/inactive-source"
+    | "capture/insufficient-storage"
+    | "capture/file-size-limit-reached"
+    | "capture/invalid-photo-codec"
+    | "capture/not-bound-error"
+    | "capture/capture-type-not-supported"
+    | "capture/video-not-enabled"
+    | "capture/photo-not-enabled"
+    | "capture/aborted"
+    | "capture/unknown";
+  declare type SystemError = "system/no-camera-manager" | "system/view-not-found";
+  declare type UnknownError = "unknown/unknown";
+
+  /**
+   * Represents a JSON-style error cause. This contains native `NSError`/`Throwable` information, and can have recursive {@linkcode ErrorWithCause.cause | .cause} properties until the ultimate cause has been found.
+   */
+  declare interface ErrorWithCause {
+    /**
+     * The native error's code.
+     *
+     * * iOS: `NSError.code`
+     * * Android: N/A
+     */
+    code?: number;
+
+    /**
+     * The native error's domain.
+     *
+     * * iOS: `NSError.domain`
+     * * Android: N/A
+     */
+    domain?: string;
+
+    /**
+     * The native error description
+     *
+     * * iOS: `NSError.message`
+     * * Android: `Throwable.message`
+     */
+    message: string;
+
+    /**
+     * Optional additional details
+     *
+     * * iOS: `NSError.userInfo`
+     * * Android: N/A
+     */
+    details?: { [key: string]: mixed, ... };
+
+    /**
+     * Optional Java stacktrace
+     *
+     * * iOS: N/A
+     * * Android: `Throwable.stacktrace.toString()`
+     */
+    stacktrace?: string;
+
+    /**
+     * Optional additional cause for nested errors
+     *
+     * * iOS: N/A
+     * * Android: `Throwable.cause`
+     */
+    cause?: ErrorWithCause;
+  }
+  declare type CameraErrorCode =
+    | PermissionError
+    | ParameterError
+    | DeviceError
+    | FrameProcessorError
+    | FormatError
+    | SessionError
+    | CaptureError
+    | SystemError
+    | UnknownError;
+  declare class CameraError<TCode: CameraErrorCode> mixins Error {
```

Comment:
is it possible to not use mixins here?

Reply:
Thanks; the "mixins" are fixed in c277db76